avb/libavb_ab/avb_ab_flow.c

```diff
@@ -152,9 +152,9 @@ static void slot_normalize(AvbABSlotData* slot) {
       /* We've exhausted all tries -> unbootable. */
       slot_set_unbootable(slot);
     }
-    if (slot->tries_remaining > 0 && slot->successful_boot) {
-      /* Illegal state - avb_ab_mark_slot_successful() will clear
-       * tries_remaining when setting successful_boot.
+    if (slot->tries_remaining > 1 && slot->successful_boot) {
+      /* Illegal state - avb_ab_mark_slot_successful() will set
+       * tries_remaining to 1 when setting successful_boot.
        */
```

Comment:
why change 0 to 1?

Reply:
because bootctrl 1.1 set tries_remaining to 1 after boot success